cns/restserver/ipam.go

```diff
@@ -34,6 +35,30 @@ const (
 	InterfaceName     = "eth0"
 )
 
+func (mcs *multicontainerstatus) String() string {
```

Comment:
What is the purpose of this method and why is the method named String()? It looks like you are appending unsuccessful error messages together for different NC IDs

Reply:
Let's change this method name to GetUnsuccessfulStatuses

Reply:
Changed the name to better reflect the purpose of the method and moved it to the same file in which the struct is defined.

Reply:
@nairashu The method as it was follows best practices. It implements `fmt.Stringer` as documented [here](https://pkg.go.dev/fmt#Stringer). This is widely recognized by many parts of the standard library that handle string-like things. For example, you could have passed this type to `fmt.Println` like this: `fmt.Println("container status:", someMcs)` and it would identify and invoke `String()`. I would recommend reverting this rename.